modelerfour/modeler/modelerfour.ts

```diff
@@ -561,11 +563,13 @@ export class ModelerFour {
       if (ei && this.interpret.isEmptyObject(ei)) {
         elementSchema = this.anySchema;
       } else {
-
+        elementNullable = (ei && ei.nullable) || false;
```

Comment:
TS has null propagation now 😄  `ei?.nullable || false`

Reply:
Old habits die hard :)  I'll simplify this, thanks!

Reply:
Ahhh, I had to do this because `.additionalProperties` is typed to `false | Schema`, so the optional chaining operator fails the type check in this case after the schema has been resolved.  It's either leave the current logic or use some type coercion to convince the compiler that it's definitely a `Schema`, so I think I'll leave this logic as-is for now.